docs/jdbc-mariadb.md

```diff
@@ -96,6 +96,59 @@ Note: a non-empty string value for the `password` property must be set. While th
 be ignored when connecting with the Cloud SQL Connector using IAM auth, leaving it empty will cause
 driver-level validations to fail.
 
+### Service Account Impersonation
+
+The Java Connector supports service account impersonation with the
+`targetPrincipal` JDBC connection property. When enabled,
+all API requests are made impersonating the supplied service account. The
+IAM principal must have the iam.serviceAccounts.getAccessToken permission or
+the role roles/iam.serviceAccounts.serviceAccountTokenCreator.
+
+Example:
+```java
+    // Set up URL parameters
```

Comment:
nit: this doesn't need to be indented with the code fence above.

Reply:
Fixed.